### DISCUSSION TITLE
Disable scrollbar markers for scope?

### DISCUSSION THREAD

**Discussion by zambetti** [zed-industries/zed#10265] [Feature Requests]
In Zed settings, the  available options to control scrollbar markers are:

```
  "scrollbar": {
    // ...
    "show": "auto",
    // Whether to show git diff indicators in the scrollbar.
    "git_diff": true,
    // Whether to show selections in the scrollbar.
    "selections": true,
    // Whether to show symbols selections in the scrollbar.
    "symbols_selections": true,
    // Whether to show diagnostic indicators in the scrollbar.
    "diagnostics": true
  },
```

Unfortunately, none of these seem to allow the two markers that show the scope to be disabled. Does anyone know if those can be disabled?

Since those two markers are visually identical to the markers for search results, they are confusing since, whenever I perform a search, it always looks like there are two more search results than there are.

**Comment by alygin** [comment]
No, unfortunately, the scope- and symbol-markers use the same `symbols_selection` setting. It doesn't look right to me too, we recently [noticed it](https://github.com/zed-industries/zed/pull/10080#pullrequestreview-1980981108).

I agree that the scrollbar settings do not reflect Zed's functionality in this part.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
